lib/omnibus/licensing.rb

```diff
@@ -253,5 +297,98 @@ def local?(license)
     def licensing_warning(message)
```

Comment:
What purpose does this special helper serve? In the rest of the code base we would just call `log.warn` directly.